### PR TITLE
Change license style from JAVADOC to SLASHSTAR_STYLE

### PR DESCRIPTION
**Pull request by junchaowu**
This will fix https://github.com/yarpc/yarpc-java/issues/104

### CODE REVIEW

**Comment by junchaowu** [comment]
@yarpc/yarpc

File: build.gradle [hunk 1]
```diff
@@ -59,6 +59,9 @@ subprojects {
         ext.year = Calendar.getInstance().get(Calendar.YEAR)
         ext.name = 'Uber Technologies, Inc'
         ignoreFailures = false
+        mapping {
+            java = 'SLASHSTAR_STYLE'
```
Comment (breerly) [inline build.gradle]: oh cool, didn't know about that.

**Comment by breerly** [comment]
:shipit:

**Comment by blampe** [comment]
heh ok

**Comment by abhinav** [comment]
thanks :shipit:

**Pull request merged**